src/integration-tests/tests/driver.rs

```diff
@@ -42,6 +42,18 @@ mod driver {
             .map_err(report)
     }
 
+    #[test_case(bigquery::client::DatasetService::builder(); "default")]
+    #[test_case(bigquery::client::DatasetService::builder().with_tracing(); "with tracing enabled")]
+    #[test_case(bigquery::client::DatasetService::builder().with_retry_policy(retry_policy()); "with retry enabled")]
```

Comment:
I think we do not need all three cases for each library. One is enough, maybe this?

```suggestion
    #[test_case(bigquery::client::DatasetService::builder().with_retry_policy(retry_policy()).with_tracing(); "with retry and tracing enabled")]
```